src/gc/gcinterface.ee.h

```diff
@@ -123,6 +123,21 @@ class IGCToCLR {
     // At the end of a background GC, gives the diagnostics code a chance to run.
     virtual
     void DiagWalkBGCSurvivors(void* gcContext) = 0;
+
+    // Informs the EE of changes to the location of the card table, potentially updating the write
+    // barrier if it needs to be updated.
+    virtual
+    void StompWriteBarrierResize(WriteBarrierResizeArgs args) = 0;
```

Comment:
The structures should be passed via pointer to avoid unnecessary copies of large structs.